pkgs/os-specific/linux/batman-adv/alfred.nix

```diff
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
   buildInputs = [ gpsd libcap libnl ];
 
   preBuild = ''
-    makeFlags="PREFIX=$out PKG_CONFIG=${pkg-config}/bin/${pkg-config.targetPrefix}pkg-config"
+    makeFlags="PREFIX=$out PKG_CONFIG=${buildPackages.pkg-config}/bin/${buildPackages.pkg-config.targetPrefix}pkg-config"
```

Comment:
```suggestion
    makeFlags="PREFIX=$out PKG_CONFIG=$PKG_CONFIG"
```
apply to both

Reply:
I instead let the makefile inherit it directly from the environment